config/locales/exceptions.en.yml

```diff
@@ -244,6 +244,12 @@ en:
           detail: "The %{size} items provided did not match the definition"
           code: 147
           status: 422
+        format:
+          <<: *defaults
+          title: Invalid format
+          detail: "'%{value}' did not match the defined format"
+          code: 148
```

Comment:
😬  sorry! I just merged code 148 as const. You might want to change this to 149.